roles/sneakers_worker/molecule/default/verify.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: test for deploy user on sudoers
+    lineinfile:
+      path: "/etc/sudoers"
+      line: "{{ item }}"
+      state: present
+    loop:
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sneakers status"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sneakers restart"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sneakers start"
+    check_mode: true
+    register: present
+    failed_when:
+      - present is changed
```

Comment:
The old tests used to check for `f = host.file('/etc/systemd/system/sneakers.service')` Is this still needed?  I see the tests for sudoers which look great, but the service itself might not exist.

Reply:
these are systemd tests that never ran before molecule3. I elected not to introduce them and issued them with #1926

Reply:
OK?